src/ios/CDVCamera.m

```diff
@@ -183,6 +208,26 @@ - (void)takePicture:(CDVInvokedUrlCommand*)command
             }
         }
 
+        if (pictureOptions.sourceType != UIImagePickerControllerSourceTypeCamera &&
+            pictureOptions.allowsEditing == NO &&
+            pictureOptions.destinationType == DestinationTypeFileUri &&
+            pictureOptions.mediaType == MediaTypePicture &&
+            pictureOptions.popoverOptions == nil) {
+            if (@available(iOS 14.0, *)) {
```

Comment:
Where is the not 14.0 branch of this condition? Is there an option to get this two logical branches in one place or more together in the call graph?

Reply:
Reason this is layed out with a shortcircuit is that @available can't be used as a BOOL in a condition. I can extract the if logic into an additional method but than we lose correct api availability checks. Alternetivly we could accept some code duplication and use a nested if else to retain api availabilty checks.